### PR TITLE
feat(framework/vue): support typescript for example/vue-app

### PR DESCRIPTION
**Pull request by linbingquan**

### CODE REVIEW

File: tests/loaders_vue_test.ts [hunk 1]
```diff
@@ -7,8 +7,8 @@ Deno.test("loaders/vue.ts: VueLoader", async () => {
   Deno.chdir(dir);
   await Deno.writeTextFile(
     join(dir, "test.vue"),
-    `
-    <script setup>
+    /* html */ `
+    <script setup lang="ts">
```
Comment (ije) [inline tests/loaders_vue_test.ts:11]: no need to add `ts` land since there is a `ts` testing existing

File: tests/loaders_vue_test.ts [hunk 2]
```diff
@@ -105,8 +105,8 @@ Deno.test("loaders/vue.ts: VueLoader(ssr)", async () => {
...
```
Comment (ije) [inline tests/loaders_vue_test.ts:109]: same

**Comment by ije** [comment]
any thoughts on vscode extension? do you think we should crate a ts server for volar with deno lsp?

**Comment by linbingquan** [comment]
> any thoughts on vscode extension?

I think if we can use volar, it is better, it can save time and volar is not bad.

> do you think we should crate a ts server for volar with deno lsp?

I feel this is the right direction. TBH, I don't have any experience with lsp.

**Review by ije — APPROVED** [review]
LGTM

**Pull request merged**